Installation Guide: Setup files (server and proxy) (#676)
SUSE/doc-susemanager: modules/installation/pages/server-setup.adoc

. Click btn:[Yes] to run setup when prompted.

> . When the setup has completed, click btn:[Next] to continue.

Reviewer: ```suggestion
. When setup is complete, click btn:[Next] to continue.
```

+

image::admin_suse_products.png[scaledwidth=80%]

. If your {sle} client is based on [systemitem]``x86_64`` architecture scroll down the page and select the check box for this channel now.
+

* Add channels to {productname} by selecting the check box to the left of each channel.
Click the arrow symbol to the left of the description to unfold a product and list available modules.

> * Start product synchronization by clicking btn:[Add Products].

Reviewer: ```suggestion
* Click btn:[Add Products] to start product synchronization.
```

> After adding the channel {productname} will schedule the channel to be synchronized.

Reviewer: ```suggestion
After adding the channel, {productname} will schedule the channel to be synchronized.
```

To update the grub2 during boot run [command]``grub2-install /dev/sda``.
====

Monitor the channel synchronization process in real-time by viewing channel log files located in the directory [path]``/var/log/rhn/reposync``:

----
tail -f /var/log/rhn/reposync/<CHANNEL_NAME>.log
----

> After the channel synchronization process has completed continue with xref:client-configuration:manual-registration-overview.adoc[Client Registration].

Reviewer: ```suggestion
When the channel synchronization process is complete, you can continue with client registration.
For more instructions, see xref:client-configuration:manual-registration-overview.adoc[].
```
